apis/phabricator.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, config):
 
     @logEntryExit
     def submit_patch(self):
-        ret = self.run(["arc", "diff", "--verbatim"])
+        ret = self.run(["arc", "diff", "--verbatim", "--conduit-uri", self.url, ".arcconfig"])
```

Comment:
Can you explain more about the `.arcconfig` included here?

Reply:
looking thru the php (because the docs are oh so helpful) this was specifying `./.arcconfig` as the config to use which it turns out is also the default config arc looks for, so this can be omitted